.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,12 @@
+name: Lint Commit Messages
+on: [pull_request]
+jobs:
+  commitlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install required dependencies
+        run: |
+          npm install @commitlint/config-conventional @commitlint/cli
```

Comment:
This step is unnecessary, cause the commitlint binary is already contains in the `wagoid/commitlint-github-action@v6`

Reply:
Thank you for your advice，I replace it to "  run: |
          npm install @commitlint/cli commitlint-config-wizardoc " ,so you can see it at "https://github.com/RAOE/CodeSnap.idea/blob/feature_commitlint_0.1/.github/workflows/commitlint.yml#L21"